graph-builder/tests/images/build-n-push-buildah.sh

```diff
@@ -15,7 +15,7 @@ for target in "${@}"; do
   tag="${tag_raw/_/-}"
 
   full_tag="${REPO_BASE}-${repo}:${tag}"
-  buildah bud -t "${full_tag}" "${target}"
+  buildah bud -t "${full_tag}" --format docker "${target}"
```

Comment:
This [graph-builder/tests/images/build-n-push-buildah.sh](https://github.com/openshift/cincinnati/blob/master/graph-builder/tests/images/build-n-push-buildah.sh) script is run in two places.
- [cincinnati/src/plugins/internal/graph_builder/dkrv2_openshift_secondary_metadata_scraper/test_fixtures/build.sh](https://github.com/openshift/cincinnati/blob/master/cincinnati/src/plugins/internal/graph_builder/dkrv2_openshift_secondary_metadata_scraper/test_fixtures/build.sh) 
- [dist/prepare_ci_images.sh](https://github.com/openshift/cincinnati/blob/master/dist/prepare_ci_images.sh)

This change is needed for the former script as described in the change's commit. However, I have not run the latter script with the new changes. I would like to run it after all tests are fixed to verify the change and to more easily notice any tests start breaking.

Just checked, it seems running the script won't be necessary probably; the existing images produced by the latter script already have the docker format.

```
$ podman manifest inspect registry.ci.openshift.org/cincinnati-ci-public/cincinnati-test-emptyfirsttag-public-manual:0.0.0 | jq .mediaType
WARN[0000] The manifest type application/vnd.docker.distribution.manifest.v2+json is not a manifest list but a single image. 
"application/vnd.docker.distribution.manifest.v2+json"
```